index.js

```diff
@@ -100,6 +100,21 @@ class PQueue {
 		return this._pendingCount < this._concurrency;
 	}
 
+	get concurrency() {
+		return this._concurrency;
+	}
+
+	set concurrency(newConcurrency) {
+		if (!(typeof newConcurrency === 'number' && newConcurrency >= 1)) {
+			throw new TypeError(`Expected \`concurrency\` to be a number from 1 and up, got \`${newConcurrency}\` (${typeof newConcurrency})`);
```

Comment:
Error should reflect parameter name:

> Expected `newConcurrency`...

Reply:
I think it should reflect the user-facing property, which is called “concurrency”. “newConcurrency” is only an internal parameter name.